### PR TITLE
WIP: Add ISO missing templates

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
*As requested in https://github.com/metanorma/mn-templates-iso/issues/2*

### CODE REVIEW

**Comment by manuelfuenmayor** [comment]
@ronaldtse , I have added technical-report template so far. In this particular case, I used [ISO/IEC TR 29166:2011(en)](https://www.iso.org/obp/ui/#iso:std:iso-iec:tr:29166:en) metadata and content to generate this template. Could I apply a similar approach on the other doctypes as well?

File: technical-report/models/datatypes.wsd [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@startuml
```
Comment (ronaldtse) [inline technical-report/models/datatypes.wsd]: Remove this file, it is irrelevant to the template.

File: technical-report/models/style.uml.inc [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@startuml
```
Comment (ronaldtse) [inline technical-report/models/style.uml.inc]: Remove this file, it is irrelevant to the template.

File: technical-report/sections/00-intro.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
+
+:sectnums!:
+[[introduction]]
+== Introduction
+
+// Insert introduction here.
+
+OASIS Open Document Format ODF 1.0 (<<iso-iec_26300>>) and Office Open XML (ISO/IEC 29500) are both open document formats for saving and exchanging word processing documents, spreadsheets and presentations. Both formats are XML based but differ in design and scope.
+
+OASIS ODF 1.0 was published by OASIS as an OASIS standard in May 2005. The second edition of ODF 1.0 has been published by OASIS as a committee specification in July 2006 and accepted as an International Standard by ISO (<<iso-iec_26300>>) in December 2006. Office Open XML was first approved in December 2006 by the ECMA International General Assembly as ECMA-376. An updated version was published in November 2008 by ISO (ISO/IEC 29500). The corresponding version, ECMA-376 2nd edition, was published in December 2008.
```
Comment (ronaldtse) [inline technical-report/sections/00-intro.adoc]: We shouldn't copy all the text here, let's only keep 1 paragraph.

File: technical-report/sections/00-intro.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+The following abbreviations are used throughout this Technical Report:
+
+* ODF, which stands for OpenDocument Format (<<iso-iec_26300>>);
+* OOXML, which stands for Office Open XML (ISO/IEC 29500:2008).
+
+It is hoped that this Technical Report will be useful in understanding how the ODF and OOXML International Standards compare and how their functionality can be mapped between the two formats. It is a necessary step to the goal of helping achieve interoperability and harmonization between the two formats.
+
+=== History of ODF and OOXML
```
Comment (ronaldtse) [inline technical-report/sections/00-intro.adoc]: Let's keep this subclause with 1 paragraph.

File: technical-report/sections/00-intro.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+ODF was originally developed by Sun Microsystems between 2000 and 2002 with the following objective:
```
Comment (ronaldtse) [inline technical-report/sections/00-intro.adoc]: Remove this until the next comment.

File: technical-report/sections/00-intro.adoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Microsoft followed suit in 2006 via the Open Specification Promise (OSP2) by opening the format of its 2007 version of the Microsoft office suite (version 12) for which it also uses XML as an exchange and storage format. OOXML is a file format originally developed by Microsoft as a successor to its earlier Office 2003 file formats. It is used for representing spreadsheet, presentation and word processing documents. In 2006 Office Open XML became an ECMA standard (ECMA-376, 1st edition). In 2008, a revised version of ECMA-376 became an ISO International Standard (ISO/IEC 29500:2008), which has its equivalent in the ECMA-376, 2nd edition.
+
+ISO/IEC 29500 is structured into four parts, each of which contains normative as well as informative material: Fundamentals and Markup Language Reference, Open Packaging Conventions, Markup Compatibility and Extensibility, and Transitional Migration Features.
+
+At the time of writing (June 2011) the following corrigenda and amendments have been published:
```
Comment (ronaldtse) [inline technical-report/sections/00-intro.adoc:16]: Keep this list.

File: technical-report/sections/01-scope.adoc [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+:sectnums:
```
Comment (ronaldtse) [inline technical-report/sections/01-scope.adoc]: We never need `:sectnums:`, remove.

File: technical-report/sections/03-termsdef.adoc [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+=== translation fidelity
+
+quality of a translation process between the ODF and OOXML document formats
+NOTE: Translation fidelity depends on document properties.
+
+NOTE: Translation fidelity cannot be measured in an absolute manner; it depends on the intentions of the document's authors.
+
+
+=== document type
```
Comment (ronaldtse) [inline technical-report/sections/03-termsdef.adoc]: Please remove the terms that have less illustrative value for the author.

File: technical-report/README.adoc [hunk 9]
```diff
@@ -0,0 +1,23 @@
+= ISO International Standard template in Metanorma
```
Comment (ronaldtse) [inline technical-report/README.adoc]: ISO Technical Report template in Metanorma

File: technical-report/data/codes.csv [hunk 10]
```diff
@@ -0,0 +1,2 @@
+﻿System code,English name,Names in other languages,Abbreviation,Specification,Remarks
```
Comment (ronaldtse) [inline technical-report/data/codes.csv]: Remove file.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: technical-report/models/datatypes.wsd (hunk 1 above)
Comment (manuelfuenmayor) [reply technical-report/models/datatypes.wsd]: Done

File: technical-report/models/style.uml.inc (hunk 2 above)
Comment (manuelfuenmayor) [reply technical-report/models/style.uml.inc]: Done

File: technical-report/sections/00-intro.adoc (hunk 3 above)
Comment (manuelfuenmayor) [reply technical-report/sections/00-intro.adoc]: Re-arranged

File: technical-report/sections/00-intro.adoc (hunk 4 above)
Comment (manuelfuenmayor) [reply technical-report/sections/00-intro.adoc]: Done

File: technical-report/sections/00-intro.adoc (hunk 5 above)
Comment (manuelfuenmayor) [reply technical-report/sections/00-intro.adoc]: Done

File: technical-report/sections/00-intro.adoc (hunk 6 above)
Comment (manuelfuenmayor) [reply technical-report/sections/00-intro.adoc:16]: Ok

File: technical-report/sections/01-scope.adoc (hunk 7 above)
Comment (manuelfuenmayor) [reply technical-report/sections/01-scope.adoc]: Removed

File: technical-report/sections/03-termsdef.adoc (hunk 8 above)
Comment (manuelfuenmayor) [reply technical-report/sections/03-termsdef.adoc]: Done

File: technical-report/README.adoc (hunk 9 above)
Comment (manuelfuenmayor) [reply technical-report/README.adoc]: Done

File: technical-report/data/codes.csv (hunk 10 above)
Comment (manuelfuenmayor) [reply technical-report/data/codes.csv]: Done

**Comment by manuelfuenmayor** [comment]
Corrections applied.
I'm following with the other templates taking into account your previous feedback. Thanks!

File: publicly-available-specification/README.adoc [hunk 11]
```diff
@@ -0,0 +1,23 @@
+= ISO International Standard template in Metanorma
```
Comment (ronaldtse) [inline publicly-available-specification/README.adoc]: Please reflect template type

File: publicly-available-specification/document.adoc [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+
+////
+Comment 4:  DOCUMENT STAGE (:docstage:) (Synonym of :status:)
+
+Stage code for the document status. See International harmonized stage codes: https://www.iso.org/stage-codes.html
+
+ISO authors usually create Metanorma documents at stages 00 through 40 (PWI, NWIP, WD, CD, DIS). Documents at the final stages, 50 and 60 (FDIS, PRF, IS), are created by ISO Editorial Program Managers at ISO/CS (aka ISO EPMs, ISO Editors). Authors using Metanorma are not expected to edit documents at those stages, and are not necessary in a normal submission process. These stages, however, can be used for mirroring and tracking of final stage and published standards, which many authors do.
+
+Example values:
+PWI: 00
```
Comment (ronaldtse) [inline publicly-available-specification/document.adoc:121]: Please see this diagram for possible stages for different deliverable types. We need to also put this on metanorma.com:

File: publicly-available-specification/sections/03-termsdef.adoc [hunk 13]
```diff
@@ -0,0 +1,48 @@
+
+[[terms]]
+== Terms and definitions
```
Comment (ronaldtse) [inline publicly-available-specification/sections/03-termsdef.adoc:3]: Add an example of `term:[...]`

File: technical-report/document.adoc [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+////
+
+////
+Comment 4:  DOCUMENT STAGE (:docstage:) (Synonym of :status:)
+
+Stage code for the document status. See International harmonized stage codes: https://www.iso.org/stage-codes.html
+
+ISO authors usually create Metanorma documents at stages 00 through 40 (PWI, NWIP, WD, CD, DIS). Documents at the final stages, 50 and 60 (FDIS, PRF, IS), are created by ISO Editorial Program Managers at ISO/CS (aka ISO EPMs, ISO Editors). Authors using Metanorma are not expected to edit documents at those stages, and are not necessary in a normal submission process. These stages, however, can be used for mirroring and tracking of final stage and published standards, which many authors do.
+
+Example values:
```
Comment (ronaldtse) [inline technical-report/document.adoc:115]: Please update to reflect:

File: technical-report/sections/03-termsdef.adoc [hunk 15]
```diff
@@ -0,0 +1,64 @@
+
+[[terms]]
+== Terms and definitions
```
Comment (ronaldtse) [inline technical-report/sections/03-termsdef.adoc:3]: Add an example to show `term:[...]`

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: publicly-available-specification/README.adoc (hunk 11 above)
Comment (manuelfuenmayor) [reply publicly-available-specification/README.adoc]: Done

File: publicly-available-specification/document.adoc (hunk 12 above)
Comment (manuelfuenmayor) [reply publicly-available-specification/document.adoc:121]: I have added additional information about possible stages and substages according to the deliverable type.

File: publicly-available-specification/sections/03-termsdef.adoc (hunk 13 above)
Comment (manuelfuenmayor) [reply publicly-available-specification/sections/03-termsdef.adoc:3]: Done.

File: technical-report/document.adoc (hunk 14 above)
Comment (manuelfuenmayor) [reply technical-report/document.adoc:115]: I have added additional information about possible stages and substages according to the deliverable type.

File: technical-report/sections/03-termsdef.adoc (hunk 15 above)
Comment (manuelfuenmayor) [reply technical-report/sections/03-termsdef.adoc:3]: Done.

**Pull request merged**